services/files.js

```diff
@@ -15,13 +15,21 @@ async function getImageTitles({ client, title, wikiUrl }) {
 
 function formatImageInfo(page) {
   const { title, imageinfo } = page;
-  const { url } = imageinfo[0];
-
-  return { title, url };
+  const {
+    url: rawUrl,
+    descriptionurl: descriptionUrl,
+    size: fileSize,
+    thumburl,
+    thumbwidth,
+    thumbheight,
+  } = imageinfo[0];
+  const thumbnail = { rawUrl: thumburl, width: thumbwidth, height: thumbheight };
+
+  return { title, descriptionUrl, rawUrl, fileSize, thumbnail };
 }
 
 async function getImageUrls({ client, titles, wikiUrl }) {
-  const params = { iiprop: 'url' };
+  const params = { iiprop: 'url|size', iiurlwidth: 300 };
```

Comment:
it's probably good to somewhere here (or in a more general readme) post a link to the documentation of these `iiprop`-style params.

https://www.mediawiki.org/wiki/API:Imageinfo

At least I wouldn't be able to always have enough of this page in memory to read and understand this code without needing to look it up :)

Reply:
I agree. The readme could probably have a "References" or "Wiki APIs" section with links to the relevant documentation. That'd be great 💚